### DISCUSSION TITLE
Welcome to eslint-plugin-playwright Discussions!

### DISCUSSION THREAD

**Discussion by mskelton** [playwright-community/eslint-plugin-playwright#88] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TomBecketBelda** [comment]
Hi I'm just installed the plugin in my playwright(I'm using it for api testing) and I don't know where to write the configuration to be used with the test runner: {
  "extends": ["plugin:playwright/playwright-test"]
}
I've tried in playwright.config.ts and in package.json but it doesn't work. Maybe you can help me with this?

**Reply by mskelton** [reply]
You'll need to add that config to your ESLint config, which usually is contained in a file named `.eslintrc`.
